org.audacityteam.Audacity.appdata.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component>
+  <id>org.audacityteam.Audacity</id>
+  <launchable type="desktop-id">org.audacityteam.Audacity.desktop</launchable>
+  <project_license>GPL-2.0 and CC-BY-3.0</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+  <name>Audacity</name>
+  <summary>Record and edit audio files</summary>
+  <description>
+    <p>Audacity® is a free, easy-to-use, multi-track audio editor and recorder for Windows, Mac OS X, GNU/Linux and other operating systems. The interface is translated into many languages.</p>
+    <p>You can use Audacity to:</p>
+    <ul>
+      <li>Record live audio</li>
+      <li>Convert tapes and records into digital recordings or CDs</li>
+      <li>Edit WAV, AIFF, FLAC, MP2, MP3 or Ogg Vorbis sound files</li>
+      <li>Cut, copy, splice or mix sounds together</li>
+      <li>Change the speed or pitch of a recording</li>
+      <li>Apply a wide range of other effects to audio recordings</li>
+    </ul>
+  </description>
+  <categories>
+    <category>AudioVideo</category>
+    <category>Audio</category>
```

Comment:
Are these not in the desktop file?

Reply:
I forgot that they are copied from the desktop file.